.changeset/silent-doors-pull.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
@timotheeguerin it seems ci not fail for this empty changelog, is it a bug?

Reply:
yeah changesets requirement for passing the CI is having a changeset file doesn't matter about the content.

We can change to use chronus here too, i just didn't do it because there was teh talk about moving it directly into typespec repo but not sure where thats' going